src/util/mq.ts

```diff
@@ -0,0 +1,17 @@
+export enum BreakPoint {
+  sm,
+  md,
+  lg,
+  xl,
+}
+
+const screenSizes = {
+  [BreakPoint.sm]: 640,
```

Comment:
(I personally think 640 is huge)

Reply:
yeah same, I've just used whatever tailwind default values were, just to keep consistency until I finally get around to removing it